agile-development/team-agreements/team-manifesto/readme.md

```diff
@@ -0,0 +1,78 @@
+# Team Manifesto
+
+## Introduction
+
+CSE teams work with a new development team in each customer engagement which requires a phase of introduction & knowledge transfer before starting an engagement.
+
+Completion of this phase of ice-breakers and discussions around the standards takes time however also required to start increasing the learning curve of the new team.
+
+A team manifesto is a soft contract among team members which summarizes the basic principles and values of the team and aiming to provide a consensus about
```

Comment:
nit: remove newline after about

Reply:
you are right
these newlines are because of my linter, it blocks me for committing without some lines/spaces. I will disable the rules :) thanks

Reply:
fixed in commit 99577bdb010e37487147d760384d41168c152334